### ISSUE TITLE
Any form should have a submit button according to WCAG2 3.2.2 H32, there is no exception

### ISSUE THREAD

**Issue by ghost** [wet-boew/wet-boew-wpss#3]
In the case of .net pages .aspx,  lots of Accessibility Tools fail the missing button problem with the Sufficient Techniques for 3.2.2 - On Input even though when the form is Hidden.

Here is the code example:
div id="cn-centre-col">--div id="cn-centre-col-inner"--
   form method="post" action="LVMap.aspx" id="form1"--
div class="aspNetHidden"--

The Total Validator 7.4.0 identifies it in 3.2.2 H32
The HiSoftware Compliance Sheriff in H32 too

Is there a particular reason why  WET-BOEW-WPSS doesn't raise this one?
